backend/inspirehep/records/marshmallow/authors/base.py

```diff
@@ -43,6 +45,9 @@ class Meta:
         exclude = FIELDS_TO_EXCLUDE
 
     email_addresses = fields.Raw(dump_only=True)
+    positions = fields.Raw(dump_only=True)
+    advisors = fields.Raw(dump_only=True)
+    project_membership = fields.Raw(dump_only=True)
```

Comment:
Just to be sure, these are defined again because on the Raw schema we have `NonHiddenRaw` right?

Reply:
yep!